.travis.yml

```diff
@@ -7,3 +7,4 @@ env:
   - secure: "BJobYIRLnVumkSlSwhTLLCFl0Li3IdpIoiuTkC39JJQzk6fyz/g86A56GXsENqwOCmWIARuZbc7F2UEMeV5Cd01P99DZdv9OC3fQUzgFEd7s+9AjYtFh91jKGEtsgC67gCxFayB9WkLiME9tkgDRNKvLRk1/9eXo776js8XG4FA="
   matrix:
   - OCAML_VERSION=latest PACKAGE=vchan DEPOPTS="mirage-xen xen-gnt xen-evtchn"
+  - OCAML_VERSION=4.03.0 PACKAGE=vchan DEPOPTS="mirage-xen xen-gnt xen-evtchn"
```

Comment:
I'd make this `latest` here explicit... to be 4.02 (which is actually not latest anymore d'oh) ;)